packages/lib/src/components/internal/SecuredFields/SecuredFieldsProvider.ts

```diff
@@ -204,6 +209,21 @@ class SecuredFieldsProvider extends Component<SFPProps, SFPState> {
     }
 
     public processBinLookupResponse(binValueObject: BinLookupObject): void {
+        console.log('### SecuredFieldsProvider::processBinLookupResponse:: binValueObject', binValueObject);
+
+        console.log('### SecuredFieldsProvider::processBinLookupResponse:: is currently in unsupportedCard state=', this.state.hasUnsupportedCard);
+        console.log('### SecuredFieldsProvider::processBinLookupResponse:: state.errors=', this.state.errors);
```

Comment:
Some more logs

Reply:
wtf! I removed all these! I actively went in and removed them because I know I'm prone to leaving log statements in the code!!!??? 
I'll fix it!